sqlite3/example/multiplatform/pubspec.yaml

```diff
@@ -0,0 +1,16 @@
+name: sqlite3_multiplatform_example
+description: Uses common interface to `sqlite3` on web and native platforms.
+version: 0.1.0
+publish_to: 'none'
+
+environment:
+  sdk: ">=2.12.0 <4.0.0"
+
+dependencies:
+  path: ^1.8.3
+  path_provider: ^2.1.1
```

Comment:
Installing the Flutter SDK in GitHub actions takes much longer than a plain Dart SDK, but it seems like we can't resolve these dependencies otherwise.

Can we comment this out as well and change the analysis step to only run on `lib/` and `test/` instead? The command in line 116 of the action could be replaced with `dart analyze --fatal-infos lib/ test/`

Reply:
Right, we need to skip CI `analyze` of example which requires `path_provider` which requires Flutter.

Done: https://github.com/simolus3/sqlite3.dart/pull/189/commits/58338bc591344d58445519e1d4608f55c3acc064